assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -101,7 +101,7 @@ function contribAttrs(isoCountry: IsoCountry, contribType: Contrib): ContribAttr
 function showPayPal(props: PropTypes) {
   if (props.contribType === 'ONE_OFF') {
     return (<PayPalContributionButton
-      amount={props.contribAmount.oneOff.value}
+      amount={Number(props.contribAmount.oneOff.value)}
```

Comment:
Is there a chance that this might crash on failure to parse?

Reply:
This was done before more deeper in the stack without checking, for example [here](https://github.com/guardian/support-frontend/pull/315/files#diff-f05d03901963ee4626fc0773bd49e0a4L38), I assumed that would continue working. I will think more about possible casting errors.

Reply:
After checking, this cast is safe. 🎉